WelcomeHome/WelcomeHome.DAL/Repositories/IEventsRepository.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore;
+using WelcomeHome.DAL.Models;
+
+namespace WelcomeHome.DAL.Repositories
+{
+    public interface IEventsRepository 
+    {
+        Task<IEnumerable<Event>> GetEventsAsync();
```

Comment:
We can not write word event in every method. It is clear from class name. GetEventsAsync -> GetAllAsync